modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/views/list/List.js

```diff
@@ -61,26 +69,54 @@ function List({datasetDisplayContext, items, schema}) {
 							/>
 						)}
 					</ClayList.ItemField>
-					{schema.thumbnail && item[schema.thumbnail] && (
+					{image && item[image] ? (
 						<ClayList.ItemField>
-							<ImageRenderer value={item[schema.thumbnail]} />
+							<ImageRenderer
+								sticker={sticker && item[sticker]}
+								value={{src: item[image]}}
+							/>
 						</ClayList.ItemField>
+					) : (
+						symbol &&
+						item[symbol] && (
+							<ClayList.ItemField>
+								<ClaySticker
+									{...((sticker && item[sticker]) ?? {})}
```

Comment:
I *think* you can make this more readable:

```
{...(sticker && item[sticker])}
```

At least, that works in normal object spreads and I imagine it will work in JSX spreads too.

The idea being that you don't need the `?? {}` fallback because spreading `null` and `undefined` is handled gracefully; eg.

```js
const x = {...null, ...undefined}; // x is now {}
```